### PR TITLE
Change webpack to microbundle and Flow types

### PR DESCRIPTION
**Pull request by stereobooster**
- add tests
- switch from webpack to microbundle
- and Flow types
- add more checks for properties
- change tabIndex to number (breaking change)
- add options to loadReCaptcha
- by default loadReCaptcha is loaded with explicit param (breaking change)

### CODE REVIEW

**Comment by stereobooster** [comment]
Ready for review

File: src/ReCaptcha.js [hunk 1]
```diff
@@ -1,120 +1,145 @@
...
+          {...rest}
+        />
+      );
 
-    return (
-      <div
-        ref={this.recaptcha}
-        data-sitekey={sitekey}
-        data-theme={theme}
-        data-type={type}
```
Comment (stereobooster) [inline src/ReCaptcha.js]: I didn't find type param in reCaptcha documentation

**Comment by narek118** [comment]
Conflicts

**Comment by stereobooster** [comment]
Not using it anymore

**Pull request closed without merging**